2-mandatory/3-stocks.js

```diff
@@ -64,10 +84,32 @@ function getPriceChanges(closingPricesForAllStocks) {
     The price should be shown with exactly 2 decimal places.
 */
 function highestPriceDescriptions(closingPricesForAllStocks, stocks) {
-    // TODO
+const result = [];
+let num = 0;
+let Name;
+for(const stocksline of closingPricesForAllStocks){
+        
+   
+         let FirstPrice = stocksline[0];
+        let i = 1;
+        while(i<stocksline.length){
```

Comment:
Same comment for the while loop as before

Reply:
while(i=1, i<stocksline.length, i++){ }